### ISSUE TITLE
Font size small on iPhone SE

### ISSUE THREAD

**Issue by lXSPandora** [gitpoint/git-point#679]
The fonts are a little bit small on the SE

**Comment by machour** [comment]
Hi and thank you for reporting this issue. 
Could you please add some screenshots of the application with the iPhone SE? That would help us a lot!

**Comment by housseindjirdeh** [comment]
Agreed - definitely think a screenshot would be super useful. Thank you for logging this @lXSPandora 🙌 

With that being said - maybe we can consider allowing the user to change between _regular_ and _large_ font sizes on the settings screen 🤔  @machour

**Comment by machour** [comment]
Good idea!

**Comment by brandly** [comment]
here's a shot of this issue:
![issue-gitpoint](https://user-images.githubusercontent.com/820696/45903439-ba79a800-bdb7-11e8-8326-988c171da6bb.png)

and here's what it looks like on github:
![issue-github](https://user-images.githubusercontent.com/820696/45903438-ba79a800-bdb7-11e8-8c63-a196b4a765c8.png)

**Comment by housseindjirdeh** [comment]
Thanks for the screenshot @brandly 🙌🏾

Definitely not a high priority, but I think a settings option to change between "Regular" and "Large" would be nice to have
